### ISSUE TITLE
Add the option to also accept other files for icons such as .jpeg .png, .gif, and especially, .ico

### ISSUE THREAD

**Issue by claudeLovelace** [FlorianWoelki/obsidian-iconize#352]
Sometime my custom icons aren't in .svg format (most if not all of theme are in .ico format) so I was wondering if there could be an option to also include other icon file types besides .svg files.

Also, the instruction on https://florianwoelki.github.io/obsidian-iconize/guide/getting-started.html#usage that instructs you to "read the documentation about custom icon packs" leads you to the starting page with no instructions.

**Comment by veitch123** [comment]
Yes. I came here to ask this too. I've got a retrobatch workflow that creates icons for me but it won't output as svg unfortunately.

**Comment by FlorianWoelki** [comment]
hey there, i've added a small section for transforming a png to svg: https://florianwoelki.github.io/obsidian-iconize/good-to-know/transform-png-to-svg.html

**Comment by KuiyueRO** [comment]
+1, I want to add some avatars of historical celebrities.

At present, I add it through [css](https://github.com/KuiyueRO/Obsidian-Miner), but it cannot cooperate with iconize.
